projects/packages/codesniffer/Jetpack/ruleset.xml

```diff
@@ -25,9 +25,6 @@
 	<rule ref="MediaWiki.WhiteSpace.EmptyLinesBetweenUse" />
 	<rule ref="MediaWiki.WhiteSpace.MultipleEmptyLines" />
 	<rule ref="MediaWiki.Usage.IsNull" />
-	<!-- Maybe in the future
```

Comment:
Maybe in another universe :)

Reply:
In this case, it's redundant to another rule. :^)